src/Traits/RequestTrait.php

```diff
@@ -158,9 +151,13 @@ protected function doStreamRequest($service, $method, $data = [])
      */
     protected function checkStatus($service, $method, $status)
```

Comment:
better rename to checkGrpcStatus, for prevent mistakes with check server status code.